lighthouse-core/lib/storage-types.js

```diff
@@ -0,0 +1,38 @@
+
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * Enum of possible storage types.
+ * @enum {string}
+ * @see https://chromedevtools.github.io/debugger-protocol-viewer/tot/Storage/#type-StorageType
+ */
+const StorageTypes = {
```

Comment:
is this file necessary? If we're only ever going to use `'all'`, it would probably be easier to use it directly